src/style.css

```diff
@@ -0,0 +1,59 @@
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+html {
+  font-weight: 300;
+  --gradient: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search_form_big {
+  margin: 20px auto;
+}
+
+.search_bar {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
```

Comment:
It'd be better to move it to `*` selector to make it universal for all of the elements
```
	*{
		box-sizing: border-box;
	}
```